### PR TITLE
Update JUNIT Tests for Bidder Seller

### PR DESCRIPTION
**Pull request by kormingsoon**

### CODE REVIEW

File: src/test/java/seedu/address/testutil/seller/TypicalSeller.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
             .withPhone("1111").withId(1).build();
     public static final Seller SELLER_ALICE = new SellerBuilder().withName("Alice Pauline")
             .withPhone("2222").withId(1).build();
-    public static final Seller BENSON = new SellerBuilder().withName("Benson Meier")
+    public static final Seller SELLER_BENSON = new SellerBuilder().withName("Benson Meier")
```
Comment (Marcon2509) [inline src/test/java/seedu/address/testutil/seller/TypicalSeller.java:24]: Why Need call it a different name?

File: src/test/java/seedu/address/logic/NextCalendarNavigationCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                new MeetingBook());
+    }
+
+    // no changes should happen to any of the books
+    @Test
+    public void execute_listIsNotFiltered_showsSameList() {
+        assertCommandSuccess(new NextCalendarNavigationCommand(), model,
+                NextCalendarNavigationCommand.MESSAGE_SUCCESS, expectedModel);
+    }
+
```
Comment (Marcon2509) [inline src/test/java/seedu/address/logic/NextCalendarNavigationCommandTest.java:42]: I dont understand what the test is for. Does navigation ever edit things in the models in the first place?

**Review by Marcon2509 — APPROVED** [review]
Got some issues can help clarify

File: src/test/java/seedu/address/logic/NextCalendarNavigationCommandTest.java (hunk 2 above)
Comment (kormingsoon) [reply src/test/java/seedu/address/logic/NextCalendarNavigationCommandTest.java:42]: Thanks, missed out the testing portion to include model inside

**Pull request merged**